modules/packages/LinearAlgebra.chpl

```diff
@@ -2250,10 +2250,10 @@ proc svd(A: [?Adom] ?t) throws
 
   // Stores singular values, sorted
   var s: [0..<min((...A.shape))] realType;
-  // Unitary matrix, U
-  var u: [0..<m, 0..<m] t;
-  // Unitary matrix V^T (or V^H)
-  var vt: [0..<n, 0..<n] t;
+  // Unitary matrix, U, inherits row offset of A
+  var u: [A.dom.ranges(0), 0..<m] t;
```

Comment:
I believe your use of `domain.ranges` should be replaced with [`domain.dim`](https://chapel-lang.org/docs/language/spec/domains.html#ChapelDomain.dim)